modules/Core/pages/panel/users_sessions.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/*
+ *  Made by Supercrafter100
+ *  https://github.com/NamelessMC/Nameless/
+ *  NamelessMC version 2.0.3
+ *
+ *  License: MIT
+ *
+ *  Panel user sessions page
+ */
+if (!$user->handlePanelPageLoad('admincp.users.sessions')) {
+    require_once(ROOT_PATH . '/403.php');
+    die();
+}
+
+const PAGE = 'panel';
+const PARENT_PAGE = 'users';
+const PANEL_PAGE = 'sessions';
+$page_title = $language->get('admin', 'user_sessions');
+require_once(ROOT_PATH . '/core/templates/backend_init.php');
+
+// Load modules + template
+Module::loadPage($user, $pages, $cache, $smarty, [$navigation, $cc_nav, $staffcp_nav], $widgets, $template);
+
+if (!isset($_GET['id']) || !is_numeric($_GET['id'])) {
+    Redirect::to(URL::build('/panel/users'));
+}
+
+$view_user = new User($_GET['id']);
+if (!$view_user->exists()) {
+    Redirect::to(URL::build('/panel/users'));
+}
+
+if (Input::exists()) {
+    if (Token::check()) {
+        if ($_POST['action'] === 'logout' && isset($_POST['sid'])) {
+            DB::getInstance()->update('users_session', $_POST['sid'], [
+                'active' => false
+            ]);
+            $success = $language->get('general', 'logout_session_successfully');
+        } else if ($_POST['action'] === 'logout_other_sessions') {
+            $view_user->logoutAllOtherSessions();
+            $success = $language->get('admin', 'sessions_logged_out');
+        }
+    } else {
+        $errors[] = $language->get('general', 'invalid_token');
+    }
+}
+$timeago = new TimeAgo(TIMEZONE);
+$sessions = $view_user->getActiveSessions();
+$user_sessions_list = [];
+
+// TODO: Should we display all sessions, or just active ones? Over time, the list could get very long if we display all sessions.
+// Not really any reason to show inactive ones, since they can't action on them.
```

Comment:
Showing inactive ones can help us find if the account has been compromised or not, Hiding sessions could miss unwanted logins